tests/factories/requests_.py

```diff
@@ -2,49 +2,49 @@
 from io import BytesIO
 
 import requests
-from factory import Factory, Faker
+from factory import Factory, Faker, post_generation
+
+# pylint:disable=no-self-argument,method-hidden
 
 
 class Response(Factory):
     class Meta:
         model = requests.Response
 
-    raw = None
-    json_data = None
     encoding = "utf-8"
     status_code = Faker(
         "random_element", elements=[200, 201, 301, 304, 401, 404, 500, 501],
     )
```

Comment:
Right so these get added as attributes directly. I think I was so confused about this, as I thought they defined the kwargs only. Obviously not, because I didn't do any work to make `status_code` appear on the object...